.gitattributes.zos

```diff
@@ -0,0 +1,22 @@
+# This .gitattributes file will cause all text files EXCEPT for
+# git's .gitattributes and .gitignore files to be encoded as EBCDIC
+# when cloned using git (from Rocket Software) on z/OS platform.
```

Comment:
Is this comment correct?  To me it looks like all files will be forced to ASCII except *.hdf and *.tdf files (and of course the binary files)

Reply:
I will update the comment.